GVFS/GVFS.Common/Maintenance/GitMaintenanceStep.cs

```diff
@@ -18,7 +21,8 @@ public GitMaintenanceStep(GVFSContext context, GitObjects gitObjects, bool requi
         }
 
         public abstract string Area { get; }
-
+        protected virtual TimeSpan TimeBetweenRuns { get; }
+        protected virtual string TimeFilePath { get; set; }

```

Comment:
I'm guessing that you left this as `virtual` so the superclass can change the `get` method. Why is `set` still here? I have two guesses (and will see it when I get down to its use later):

1. You don't need it, and can delete it.
2. You need it, but you could change the protection to `{ get; protected set; }`.

Reply:
(I now see it is already `protected` so the second option is silly.)

Reply:
@derrickstolee good questions!

These should really be abstract and extended by each class same as Area.  But I don't want to force each class to implement it until we do the refactoring in #617.  I left a comment on the issue ticket to address these.